src/ImageSharp/Formats/Tiff/TiffDecoderCore.cs

```diff
@@ -71,6 +60,8 @@ public TiffDecoderCore(Configuration configuration, ITiffDecoderOptions options)
         /// </summary>
         public ushort[] BitsPerSample { get; set; }
 
+        public int ChunkyBitsPerPixel { get; set; }
```

Comment:
I do not understand the introduction of `ChunkyBitsPerPixel `. Is it not the same as `BitsPerSample`?

Reply:
different `PlanarConfigurations` (Planar and Chunky) use a different `BitsPerPixel` for data storage, see [CalculateStripBufferSize](https://github.com/SixLabors/ImageSharp/pull/1471/files/7cdd77d75bd14e3262c07fd6fb1bbf7ac0d37f1b#diff-7aee87e7eb3fd59490713053c0420b1f47f02dfdd583fddc81322bfc9dcae344R239), but the [DecodeStripsPlanar ](https://github.com/SixLabors/ImageSharp/pull/1471/files/7cdd77d75bd14e3262c07fd6fb1bbf7ac0d37f1b#diff-7aee87e7eb3fd59490713053c0420b1f47f02dfdd583fddc81322bfc9dcae344R273) method use `BitsPerPixel` as `Chunky` probably for `prdector`. To highlight this discrepancy, I named the variable `ChunkyBitsPerPixel `